_data/glossary.yaml

```diff
@@ -219,6 +221,9 @@ parent image: |
   in the image's Dockerfile. All subsequent commands are applied to this parent
   image. A Dockerfile with no `FROM` directive has no parent image, and is called
   a **base image**.
+persistent storage: |
+  Persistent storage or volume storage provides ways for an application or user to
+  store data generated by a running container. It extends beyond the life or boundaries of an existing or running container.
```

Comment:
I would argue that not all Data is generated by the running container. You could create a Volume from an existing directory on the host. Or from a Volume that was created by a previously ran container.

`Persistent storage or volume storage provides a way for a user to add a persistent layer to the running container's file system. This persistent layer could live on the container host or an external device. The lifecycle of this persistent layer is not connected to the life cycle of the container. Allowing a user to retain state.`